src/features/_marketingAndCommunication/config/deeplinksExportConfig.ts

```diff
@@ -49,21 +51,21 @@ export const SCREENS_CONFIG: Partial<Record<ScreenNames, ScreenConfig>> = {
 export const MARKETING_CONFIG: Record<string, paramConfig> = {
   utm_campaign: {
     type: 'string',
-    description: `Permet d'identifier le nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
+    description: `Nom de la campagne relative au produit, son slogan ou encore son code promotionnel.`,
   },
   utm_source: {
     type: 'string',
-    description: `Permet d'identifier, entre autres, l'annonceur, le site et la publication générant du trafic`,
+    description: `Annonceur, le site ou publication générant du trafic`,
```

Comment:
Si tu retires aux autres noms il faut aussi retirer celui-là
```suggestion
    description: `Annonceur, site ou publication générant du trafic`,
```

Reply:
Bien vu !